### ISSUE TITLE
Poor Color Contrast of the Example Pages

### ISSUE THREAD

**Issue by jfc3-dol** [mdn/content#25423]
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals

### What specific section or headline is this issue about?

Active learning: More color choices

### What information was incorrect, unhelpful, or incomplete?

The light gray text ( #A9A9A9 ) and yellow background  ( #FFFF00 ) on the Yellow and Gray example on this page and the the example page does not have enough color contrast.

The color contrast ratio is 2.2:1 and needs to be 4.5:1 or higher to pass for normal text and 3.0:1 for larger text.

Please see the attached screenshot.

<img width="732" alt="mdn - conditionals - poor color contrast" src="https://user-images.githubusercontent.com/10423487/225931585-2f529d0c-6c9e-4c2c-800b-a6ab7185feba.png">

### What did you expect to see?

I expected the examples used would have the correct color contrast to show people what is expected.

### Do you have any supporting links, references, or citations?

no

### Do you have anything more you want to share?

no

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript/building_blocks/conditionals`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/building_blocks/conditionals/index.md
* Last commit: https://github.com/mdn/content/commit/8e2641ebe076ab89299c77a51ece882de4ba5efb
* Document last modified: 2023-02-23T23:55:46.000Z

</details>
